java-client/barrage-examples/src/main/java/io/deephaven/client/examples/SubscribeExampleBase.java

```diff
@@ -42,7 +43,9 @@ protected void execute(final BarrageSession client) throws Exception {
 
         try (final TableHandle handle = manager.executeLogic(logic());
                 final BarrageSubscription subscription = client.subscribe(handle, options)) {
-            final BarrageTable table = subscription.entireTable();
+
+            final BarrageTable table = subscription.partialTable(RowSetFactory.flat(10), null, true);
```

Comment:
It might be nicer to have a `-t` command line option instead - to mention how many rows from the tail you would like to subscribe to. It's probably nice to keep an example that does subscribe to the entire table.

Reply:
Added `--tail` and  `--head` command line args to support CLI viewports.